docker/docker-compose.yml

```diff
@@ -143,6 +143,8 @@ services:
     depends_on:
       kafka-zookeeper:
         condition: service_healthy
+      schema-registry:
```

Comment:
This is a painful one and some time is taking quite long. This we could have kept service_started only.

Reply:
preferring reliable over execution time!

Reply:
Good call!

Reply:
This was not due to the improvement of execution time. This was done for reliability only. The docker-compose setup has a limitation of 60 seconds for the dependency service to be healthy. If it doesn't become healthy, it marks the dependent container has un-healthy and fails. So, to support in low configuration, either we should use a solution like wait-for-it or wait only for service_started. we choose later and waited for the successful start of required data services in hypetrace-service. Once we do the improvement of hypertrace-ingester, we should only wait for kafka-zookeeper as service_healthy.